include/deal.II/fe/fe.h

```diff
@@ -1356,11 +1356,28 @@ class FiniteElement : public Subscriptor, public FiniteElementData<dim>
    * For a definition of domination, see FiniteElementDomination::Domination
    * and in particular the
    * @ref hp_paper "hp paper".
+   *
+   * @deprecated Use compare_for_domination() with `codim=1` instead.
    */
-  virtual FiniteElementDomination::Domination
+  DEAL_II_DEPRECATED virtual FiniteElementDomination::Domination
   compare_for_face_domination(
     const FiniteElement<dim, spacedim> &fe_other) const;
 
+  /**
+   * Return whether this element dominates a neighboring one given as argument,
```

Comment:
They don't actually have to be neighbors. They typically are, but it's not necessary and immaterial for this function.